gcloud-java-core/src/main/java/com/google/gcloud/BasePage.java

```diff
@@ -22,9 +22,9 @@
 import java.util.Objects;
 
 /**
- * Base implementation for Google Cloud storage list result.
+ * Base implementation for Google Cloud paginated results.
  */
-public class BaseListResult<T extends Serializable> implements ListResult<T>, Serializable {
+public class BasePage<T extends Serializable> implements Page<T>, Serializable {
 
   private static final long serialVersionUID = -6937287874908527950L;
```

Comment:
It does not *have* to change (what really matters is that value is the same for all backward compatible versions) but I agree that it would be nice to start with the value that would be automatically generated if `serialVersionUID` was missing.